tests/vufind_templates.php-cs-fixer.php

```diff
@@ -5,34 +5,26 @@
     ->name('*.phtml');
 
 $rules = [
+    '@PSR12' => true,
     'align_multiline_comment' => true,
     'array_syntax' => ['syntax' => 'short'],
     'binary_operator_spaces' => [
         'default' => 'single_space',
     ],
-    'blank_line_after_namespace' => true,
-    //'braces' => true, // disabled because we don't want to create inconsistent indentation, but useful to normalize control structure spacing
+    'blank_line_after_opening_tag' => false,
+    'braces' => false, // disabled because we don't want to create inconsistent indentation, but useful to normalize control structure spacing
     'cast_spaces' => ['space' => 'none'],
     'class_attributes_separation' => ['elements' => ['method' => 'one', 'property' => 'one']],
     'concat_space' => ['spacing' => 'one'],
-    'constant_case' => ['case' => 'lower'],
-    'elseif' => true,
-    'encoding' => true,
     'ereg_to_preg' => true,
-    'full_opening_tag' => true,
-    'function_declaration' => true,
     'function_typehint_space' => true,
-    'indentation_type' => true,
     'is_null' => true,
-    'line_ending' => true,
     'list_syntax' => ['syntax' => 'short'],
     'lowercase_cast' => true,
-    'lowercase_keywords' => true,
     'magic_constant_casing' => true,
-    'method_argument_space' => true,
     'native_function_casing' => true,
     'no_blank_lines_after_class_opening' => true,
-    'no_blank_lines_after_phpdoc' => true,
+    'no_closing_tag' => false,
```

Comment:
Do we really need this? I believe it's useful in ensuring that a file doesn't output unexpected line feeds etc.

Reply:
In templates, we might want to output whitespace, and it makes things look unbalanced when, e.g. a template ends with `<? endif`.

Reply:
Understood, but the rule only applies to files that contain only PHP, so you'd need to have something like `echo "\n";` in place anyway. Otherwise you'd depend on something very unobvious.

Reply:
Do you have time to change the rule and run the fixer? I don't think php-cs-fixer is smart enough to restrict the rule to PHP-only files, and I'm pretty sure it messed up a lot of files, but maybe I'm wrong. I can do it in a couple hours to double check, but I'm not in the office yet.

Reply:
Tested, and to me it looks like all the changes make sense. Removing the closing tags also allows adding of ending line feeds, which is a good thing.

Reply:
You're right, I think I was blaming the `no_closing_tag` rule for problems that were actually being caused by the `braces` rule. Thanks for looking into it more carefully than I did! :-)

I've also put a comment on `no_trailing_whitespace_in_comment` to document the reason we're disabling that rule.